frontend/src/stores/ParticipantStore.js

```diff
@@ -306,6 +316,48 @@ export class ParticipantStore {
       throw `ParticipantStore:  updateVisit() Failed  =>  ${error}`
     }
   })
+  getSites = flow(function*() {
+    try {
+      const { ok, data } = yield api.getSites()
+      if (!ok || !data) {
+        throw "a placeholder error string!"
+      }
+      this.setSites(data)
+    } catch (error) {
+      throw `ParticipantStore:  getSites() Failed  =>  ${error}`
+    }
+  })
+  createSEP = flow(function*({
+    program,
+    urgency,
+    participant,
+    needles_in,
+    needles_out,
+    site,
+  }) {
+    try {
+      //{"participant":["Incorrect type. Expected pk value, received str."]}
+      const { visitOk, visitData } = yield api.createVisits({
```

Comment:
This is going to fail, because they are not apiSauce keys
to destructure and rename do this
` const { ok: visitOk, data: visitData } = yield api.createVisits({`

Reply:
Thanks. After the latest round of changes I'm getting the following error on the createVisits api call:

AttributeError at /api/visits/
'NoneType' object has no attribute 'program'

Reply:
that's a weird error. its actually because it's trying to find service. service has program as a foreign key in the db. null object has no program fk. include service ID in Post

Reply:
Thanks. That fixed it.